docs/PORTABLE_SERVICES.md

```diff
@@ -174,7 +174,12 @@ requirements are made for an image that can be attached/detached with
    recommended to use these two paths.)
 
 4. The image must contain an os-release file, either in `/etc/os-release` or
-   `/usr/lib/os-release`. The file should follow the standard format.
+   `/usr/lib/os-release`. The file should follow the standard format. In
+   addition, `portablectl` also lets you set an extra variable
+   `PORTABLE_PRETTY_NAME` in that file, which will be printed if you run
```

Comment:
better to add this to the bottom of the `Requirements on Images` section which already talks about PORTABLE_PREFIXES